opendevin/runtime/utils/runtime_templates/Dockerfile.j2

```diff
@@ -61,6 +60,7 @@ RUN cd /opendevin/code && \
     /opendevin/miniforge3/bin/mamba run -n base poetry run pip install playwright && \
     /opendevin/miniforge3/bin/mamba run -n base poetry run playwright install --with-deps chromium && \
     /opendevin/miniforge3/bin/mamba run -n base poetry cache clear --all . && \
+    {% if not skip_init %}chmod -R g+rws /opendevin/poetry && {% endif %} \
```

Comment:
Otherwise `%pip install XXX` run by non-root user inside jupyter-session will not work due to permission issue. Since we are already operate in the sandbox, i think we should be fine?